File: Plov/src/Rice.java

public class Rice {
	
	private int ready;
	
public int Ready(){
		
		return ready;
	}

	   private boolean dirty = true;
	    public boolean GetDirty (){
	    	return dirty;
	    }
	    public void SetDirty(boolean d){
	    	dirty=d;
	    }
	    
	    private boolean add = false;
	    public boolean GetAdd (){
	    	return add;
	    }
	    public void SetAdd(boolean a){
	    	add=a;
	    }
	    
	    public void Cook()
	    {
	        if (Ready() < 10)
	        {
	            ready++;
	   
	        }
	    }
	
	
}

File: Laba1Plov/Laba1Plov/Rice.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba1Plov
{
    class Rice
    {

        private bool dirty = true;
        public bool Dirty { set { dirty = value; } get { return dirty; } }

        private int ready;
        public int Ready { get { return ready; } }

        public void Cook()
        {
            if (Ready < 10)
            {
                ready++;

            }
        }
    }
}
